### ISSUE TITLE
Three potential tiers of customization/personalization

### ISSUE THREAD

**Issue by Splitwirez** [waliarubal/Jaya#85]
Lowest tier: Custom colour scheme + header image
Middle tier: Custom XAML skinning
Highest tier: Custom front-end

...I guess these would probably be implemented in post-release updates from smallest to largest (if at all)?

**Comment by waliarubal** [comment]
That's correct, you have outlined the tiers correctly. For v1 it would be a lot of tasks to deal with, we can work things up in coming releases.

**Comment by waliarubal** [comment]
This is dependent on #75

**Comment by Splitwirez** [comment]
![image](https://i.imgur.com/v0Z5Zsa.png)
Just leaving this here as food for thought for the long run
